lookout/core/metrics.py

```diff
@@ -1,12 +1,197 @@
+import os
+import string
+from threading import Lock
 from typing import Union
 
+from prometheus_client import start_http_server
+from prometheus_client.metrics import MetricWrapperBase
 
-def submit_event(key: str, value: Union[int, float, bool]):
+
+PROMETHEUS_SERVER = None
+
+PROMETHEUS_HOST = os.getenv("PROMETHEUS_HOST", "0.0.0.0")
+
+PROMETHEUS_PORT = int(os.getenv("PROMETHEUS_PORT", "8000"))
+
+
+class MutexValue:
+    """A float protected by a mutex."""
+
+    _multiprocess = False
```

Comment:
```suggestion
```

Reply:
This is needed for prometheus to work correctly